### PR TITLE
Update uml docu and readme links

### PR DESCRIPTION
**Pull request by Karle3**
UML diags added.

### CODE REVIEW

File: doc/uml/ArduinoNative.plantuml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+@startuml ArduinoNative
+
+' --- title ArduinoNative ClassDiagramm
```
Comment (BlueAndi) [inline doc/uml/ArduinoNative.plantuml]: Please remove the comment.

File: doc/uml/DynamicFlow.plantuml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            Arduino <-- App
+
+        main <-- Arduino
+    end loop
+
+
+@enduml
+
+
+
```
Comment (BlueAndi) [inline doc/uml/DynamicFlow.plantuml]: Remove last empty lines.

File: doc/uml/Principle.plantuml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+@startuml Principle
```
Comment (BlueAndi) [inline doc/uml/Principle.plantuml:1]: The principle doesn't match with ArduinoNative. Please replace at all.

File: examples/example/platformio.ini [hunk 4]
```diff
@@ -1,4 +1,4 @@
-;PlatformIO Project Configuration File
```
Comment (BlueAndi) [inline examples/example/platformio.ini]: Please revert file.

**Review by BlueAndi — CHANGES_REQUESTED** [review]

File: doc/uml/DynamicFlow.plantuml (hunk 2 above)
Comment (Karle3) [reply doc/uml/DynamicFlow.plantuml]: done

**Review by BlueAndi — APPROVED** [review]

**Comment by BlueAndi** [comment]
#1

**Pull request merged**